src/interfaces/ISablierMerkleBase.sol

```diff
@@ -97,4 +109,13 @@ interface ISablierMerkleBase is IAdminable {
     /// @param factoryAdmin The address of the `FACTORY` admin.
     /// @return feeAmount The amount of native tokens (e.g., ETH) collected as fees.
     function collectFees(address factoryAdmin) external returns (uint256 feeAmount);
+
+    /// @notice Sets the minimum fee to a lower value.
```

Comment:
`/// @notice Sets the minimum fee to a new value.`

That new fee should be less than `minimumFee` is a part of the requirement.

Reply:
I agree its a lower value but its still provided as the input of function. So how about?

`Sets the minimum fee to a new value lower than the current fee`